### ISSUE TITLE
Wingpanel failed at the compile phase

### ISSUE THREAD

**Issue by dragynbane222** [pimvullers/elementary#227]
Wingpanel fails with the following errors:

FAILED: wingpanel-interface/libwingpanel-interface.so.p/Main.c wingpanel-interface/libwingpanel-interface.so.p/DBusServer.c wingpanel-interface/libwingpanel-interface.so.p/BackgroundManager.c wingpanel-interface/libwingpanel-interface.so.p/FocusManager.c wingpanel-interface/libwingpanel-interface.so.p/Utils.c wingpanel-interface/wingpanel-interface.h wingpanel-interface/wingpanel-interface.vapi 
/usr/bin/valac-0.56 -C --vapidir=/var/tmp/portage/pantheon-base/wingpanel-3.0.2-r1/work/wingpanel-3.0.2/vapi --pkg mutter-clutter-10 --pkg mutter-cogl-pango-10 --pkg mutter-cogl-10 --pkg libmutter-10 --pkg granite --pkg gala --color=always --directory wingpanel-interface/libwingpanel-interface.so.p --basedir ../wingpanel-3.0.2/wingpanel-interface --library wingpanel-interface --header wingpanel-interface/wingpanel-interface.h --vapi ../wingpanel-interface.vapi --define HAS_MUTTER338 --define HAS_MUTTER40 --define HAS_MUTTER41 --define HAS_MUTTER42 ../wingpanel-3.0.2/wingpanel-interface/Main.vala ../wingpanel-3.0.2/wingpanel-interface/DBusServer.vala ../wingpanel-3.0.2/wingpanel-interface/BackgroundManager.vala ../wingpanel-3.0.2/wingpanel-interface/FocusManager.vala ../wingpanel-3.0.2/wingpanel-interface/Utils.vala
gala.vapi:93.10-93.22: error: The type name `Clutter.Grab' could not be found
   93 | 		public Clutter.Grab? grab { get; set; }
      | 		       ^~~~~~~~~~~~~                   
gala.vapi:93.10-93.22: error: The type name `Clutter.Grab' could not be found
   93 | 		public Clutter.Grab? grab { get; set; }
      | 		       ^~~~~~~~~~~~~                   
gala.vapi:93.10-93.22: error: The type name `Clutter.Grab' could not be found
   93 | 		public Clutter.Grab? grab { get; set; }
      | 		       ^~~~~~~~~~~~~      
      
  I'll upload the entire log as well, just in case it's needed:
[build.log](https://github.com/pimvullers/elementary/files/9081178/build.log)

**Comment by pimvullers** [comment]
Could you please send me an overview which clutter packages you have installed (and with which USE flags)?

For example using following command:
```
$ eix -I clutter
[I] media-libs/clutter
     Available versions:  (1.0) 1.26.4-r1^t
       {X aqua debug doc egl gtk +introspection test wayland}
     Installed versions:  1.26.4-r1(1.0)^t(01:36:03 PM 06/28/2022)(X egl gtk introspection wayland -aqua -debug -doc -test)
     Homepage:            https://wiki.gnome.org/Projects/Clutter
     Description:         Clutter is a library for creating graphical user interfaces

[I] media-libs/clutter-gst
     Available versions:  (3.0) 3.0.27-r1 3.0.27-r2
       {X debug examples +introspection udev}
     Installed versions:  3.0.27-r2(3.0)(04:18:16 PM 07/08/2022)(X introspection udev -debug)
     Homepage:            https://blogs.gnome.org/clutter/
     Description:         GStreamer integration library for Clutter

[I] media-libs/clutter-gtk
     Available versions:  (1.0) 1.8.4-r1
       {X debug examples gtk +introspection wayland}
     Installed versions:  1.8.4-r1(1.0)(01:26:17 AM 07/01/2022)(X gtk introspection wayland -debug -examples)
     Homepage:            https://wiki.gnome.org/Projects/Clutter
     Description:         Library for embedding a Clutter canvas (stage) in GTK+

Found 3 matches
```

**Comment by pimvullers** [comment]
3.0.2-r2 available which includes the fix for this build failure.
